ethers-middleware/tests/it/transformer.rs

```diff
@@ -0,0 +1,120 @@
+use crate::{get_wallet, spawn_anvil};
+use ethers_contract::abigen;
+use ethers_core::{abi::AbiEncode, types::*};
+use ethers_middleware::{
+    transformer::{ds_proxy::factory::DsProxyFactory, DsProxy, TransformerMiddleware},
+    MiddlewareBuilder, SignerMiddleware,
+};
+use ethers_providers::{Http, Middleware, Provider};
+use ethers_signers::{LocalWallet, Signer};
+use rand::Rng;
+use std::sync::Arc;
+
+type HttpWallet = SignerMiddleware<Provider<Http>, LocalWallet>;
+
+abigen!(SimpleStorage, "../tests/testdata/SimpleStorage.json");
+
+#[tokio::test]
+#[ignore]
```

Comment:
This single test was not ignore before, but after using abigen the final assertions fail. I don't know how to fix it because I don't know what a DsProxy even is lol

Reply:
https://github.com/dapphub/ds-proxy we should understand why it's failing